docs(portal) session updates (#1328)
Kong/docs.konghq.com: app/enterprise/0.35-x/developer-portal/configuration/authentication/key-auth.md

>**Warning** Enabling authentication in the Dev Portal requires use of the
> Sessions plugin. Developers will not be able to login if this is not set
> properly. More information about [Sessions in the Dev Portal](/enterprise/{{page.kong_version}}/developer-portal/configuration/authentication/sessions)

### Enable Portal Session Config

```
portal_session_conf={ "cookie_name": "portal_session", "secret": "CHANGE_THIS", "storage": "kong" }
```

> For testing purposes when using HTTP, example config should include `"cookie_secure": false`:

Reviewer: ```suggestion
If using HTTP while testing, include `"cookie_secure": false` in the config:
```

Reviewer 2: (same note as key auth)
